apps/web-ext/src/components/siteProfile/SiteProfile.tsx

```diff
@@ -10,76 +10,84 @@ import placeholderLogoMainUrl from "@originator-profile/ui/src/assets/placeholde
 import ReliabilityGuide from "../ReliabilityGuide";
 import WebMediaProfileSummaryCard from "../WebMediaProfileSummaryCard";
 import { SiteProfileProps } from "./types";
+import GlobalHeader from "../GlobalHeader";
 
 export function SiteProfile(props: SiteProfileProps) {
   const dialog = useModalDialog();
   return (
-    <div data-testid="site-profile" className="bg-gray-50 p-4">
-      <div className="flex justify-center">
-        <ModalDialog
-          dialogRef={dialog.ref}
-          className="group-aria-hidden:translate-y-full translate-y-0 bottom-0 w-full"
-          onClose={dialog.close}
-        >
-          <TechInfo
-            className="rounded-b-none max-h-[80svh] overflow-auto"
-            value={props.siteProfile}
-          />
-        </ModalDialog>
-        <p className="whitespace-pre-line jumpu-badge bg-gray-600 text-xs text-white font-normal border border-gray-300 mb-3">
-          {_("ContentType_Site")}
-        </p>
-        <button
-          className="jumpu-icon-button text-xs rounded-full bg-gray-100 border-gray-200 w-6 h-6 ml-1 group"
-          aria-describedby="tooltip-1"
-          onClick={dialog.open}
-        >
-          <Icon className="inline" icon={"fa6-solid:wrench"} />
-          <span
-            id="tooltip-1"
-            role="tooltip"
-            className="[transform:translate(-50%,150%)_scale(0)]! group-hover:[transform:translate(-50%,150%)_scale(1)]!"
-          >
-            {_("SiteProfile_TechnicalInformation")}
-          </span>
-        </button>
-      </div>
-      <div className="flex flex-col items-center gap-4">
-        {props.wsp.credentialSubject.image && (
-          <Image
-            className="shrink-0 w-fit"
-            src={props.wsp.credentialSubject.image.id}
-            placeholderSrc={placeholderLogoMainUrl}
-            alt=""
-            width={240}
-            height={44}
-          />
+    <>
+      <GlobalHeader className="sticky top-0 z-10">
```

Comment:
z-index を 11 以上にしないと次の箇所と同じスタッキングコンテキストで下に重なってしまうみたい

https://github.com/originator-profile/originator-profile/blob/856535bac41b355f2bf04332bf098798feb20215/apps/web-ext/src/components/credentials/Credentials.tsx#L71-L73

```suggestion
      <GlobalHeader className="sticky top-0 z-11">
```